code/modules/projectiles/guns/projectile/automatic/atreides.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/weapon/gun/projectile/automatic/atreides
 	name = "FS SMG .35 Auto \"Atreides\""
-	desc = "The Atreides is a replica of an old and popular SMG. It has a strong kick. Uses .35 Auto rounds."
+	desc = "The Atreides is a replica of an old and popular SMG. Cheap and mass produced generic self-defence weapon. \
+			Design is so generic that all of it characteristics is neither too bad or too good. Uses .35 Auto rounds."
```

Comment:
"The overall design is so generic that it is neither considered good nor bad in comparison to other firearms. Uses .35 Auto rounds."

Reply:
Many thanks, implemented.